src/store/modules/webhook/actions.ts

```diff
@@ -36,30 +36,13 @@ const actions: ActionTree<WebhookState, RootState> = {
     }
   },
   async subscribeWebhook({ dispatch }, id: string) {
-
-    // stores the webhook service that needs to be called on the basis of current webhook selected, doing
-    // so as we have defined separate service for different webhook subscription
-    const webhookMethods = {
-      'NEW_ORDERS': WebhookService.subscribeNewOrderWebhook,
-      'CANCELLED_ORDERS': WebhookService.subscribeCancelledOrderWebhook,
-      'PAYMENT_STATUS':WebhookService.subscribePaymentStatusWebhook,
-      'RETURNS': WebhookService.subscribeReturnWebhook,
-      'NEW_PRODUCTS': WebhookService.subscribeNewProductsWebhook,
-      'DELETE_PRODUCTS': WebhookService.subscribeDeleteProductsWebhook,
-      'BULK_OPERATIONS_FINISH': WebhookService.subscribeFileStatusUpdateWebhook
-    } as any
-    const webhookMethod: any = webhookMethods[id];
-
-    if (!webhookMethod) {
-      showToast(translate("Configuration missing"));
-      return;
-    }
+    const webhookMethod: any = WebhookService.webhookMethods[id];
 
     let resp;
 
     try {
-      resp = await webhookMethod({ shopifyConfigId: this.state.user.shopifyConfig })
-
+      resp = await webhookMethod({ shopifyConfigId: this.state.user.shopifyConfig }, id)
```

Comment:
I think we need to call the subscribe webhook action.

Reply:
Done sir!